### PR TITLE
mkfifo: add more information link

### PR DESCRIPTION
**Pull request by marchersimon**
For #5510

### CODE REVIEW

File: pages.bs/common/mkfifo.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # mkfifo
 
 > Pravi FIFOs (imenovane cijevi).
+> ? <https://www.gnu.org/software/coreutils/manual/html_node/mkfifo-invocation.html>.
```
Comment (bl-ue) [inline pages.bs/common/mkfifo.md:4]: 👀 !

Comment (bl-ue) [reply pages.bs/common/mkfifo.md:4]: @Vukory can we have the most accurate Bosnian translation of "More information:" ?

Comment (marchersimon) [reply pages.bs/common/mkfifo.md:4]: Yeah, I'm searching for the correct translation at the moment :sweat_smile:

Comment (bl-ue) [reply pages.bs/common/mkfifo.md:4]: Would `Više informacija` work?

Comment (marchersimon) [reply pages.bs/common/mkfifo.md:4]: I think `Više informacija:` should be correct

Comment (bl-ue) [reply pages.bs/common/mkfifo.md:4]: Let's do it.
```suggestion
> Više informacija: <https://www.gnu.org/software/coreutils/manual/html_node/mkfifo-invocation.html>.
```
I've also updated [`set_more_info_link.py`](https://gist.github.com/bl-ue/5b52c9f80dd21074c0633a2bcdeb01fb).

Comment (marchersimon) [reply pages.bs/common/mkfifo.md:4]: Also, `da` is `Mere information:` and `no` is `Mer informasjon:`

Comment (bl-ue) [reply pages.bs/common/mkfifo.md]: Oh, nice! I'll add those.

Comment (marchersimon) [reply pages.bs/common/mkfifo.md]: Thanks!

**Review by yutyo — APPROVED** [review]
LGTM.

**Review by navarroaxel — APPROVED** [review]

File: pages.bs/common/mkfifo.md (hunk 1 above)
Comment (Vukory) [reply pages.bs/common/mkfifo.md]: > Would `Više informacija` work?
`Više informacija` is perfect, though I see I'm a bit late sorry, been a rough day ^^'

Comment (bl-ue) [reply pages.bs/common/mkfifo.md]: No problem, thank you very much for the confirmation! ❤️

**Pull request merged**